src/applications/discharge-wizard/components/v2/ReviewPage.jsx

```diff
@@ -43,15 +47,13 @@ const ReviewPage = ({ formResponses, router, viewedIntroPage }) => {
             className="vads-u-margin-bottom--0 vads-u-padding-y--3 vads-u-padding-x--1p5 answer-review"
           >
             {reviewLabel}
-            {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
-            <a
+            <va-link
```

Comment:
We can't upgrade this to a `va-link` yet because the `aria-label` does not apply to the `<a>` tag.

<img width="284" alt="Screenshot 2024-07-08 at 9 02 44 AM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/19175324/4c6e6cde-1e10-43e3-8471-aaae984d8d79">

You can also see from this screenshot that the `className` is not being applied. I can't recall if there's a workaround for this; when we can convert to a web component you may need to apply this styling to something else around it.

Reply:
@randimays and @chriskim2311 this is 50% accurate - `aria-label` doesn't work because it's not the correct prop, the prop is just `label`. However, the Review screen is not finalized and TBH I don't know what in the world I want to do with these because a link labelled "Edit Did you complete a period of service in which your character of service was Honorable or General Under Honorable Conditions?" is so insane.

What I'm saying is the correct prop should probably be used but I will most likely have further edits to the Review page later on.

Reply:
Thank you @laflannery for the life of me I can never remember what is the correct state of `va-link`. We should probably update [this guidance doc](https://docs.google.com/document/d/1qMWUZw0UbumWFCRXA32BnwTYMJCK6jEv0n5hcNhnxDs/edit#heading=h.k59dbwfeoffd) though.

Reply:
@randimays sorry about that! Updated

Reply:
@laflannery What would be the proper mark up for this as an <a/> I am getting some errors here due to the onClick handler that was implemented. 
![ReviewPage_jsx_—_Code_Repos](https://github.com/department-of-veterans-affairs/vets-website/assets/42885441/9b757b87-0dd7-4883-8cec-ae06ea5c5228)

There will be another ticket that handles review page functionality and logic but for now what would be the proper solution here?

Reply:
@chriskim2311 for now, I think you can just remove the additional `label` because eventually we would be added the questions in bold and this is where the text will come from (like we do on Income limits). Let me know if this helps/resolve the error.
Screen shot for reference:
![image](https://github.com/department-of-veterans-affairs/vets-website/assets/106678594/5a7bbf1f-31ed-45af-bc96-617ec81f63b1)



So all the edits links will be:
`<va-link name="{question variable name}" text="Edit" class="hydrated vads-u-padding-left--2"></va-link>`

(also FWIW, Randi had mentioned above that `classname` isn't being applied - every other component used just the typical `class` attribute so I added that into the snippet above)